src/gentropy/method/colocalisation.py

```diff
@@ -188,12 +188,10 @@ def colocalise(
                 .withColumn("lH2bf", f.log(f.col("priorc2")) + f.col("logsum2"))
                 # h3
                 .withColumn("sumlogsum", f.col("logsum1") + f.col("logsum2"))
-                # exclude null H3/H4s: due to sumlogsum == logsum12
-                .filter(f.col("sumlogsum") != f.col("logsum12"))
                 .withColumn("max", f.greatest("sumlogsum", "logsum12"))
                 .withColumn(
                     "logdiff",
-                    (
+                    f.when(f.col("sumlogsum") == f.col("logsum12"), 1e-10).otherwise(
```

Comment:
You could specify `PSEUDOCOUNT:float=1e-10` at the beginning of the class.  
```suggestion
                    f.when(f.col("sumlogsum") == f.col("logsum12"), ECaviar.PSEUDOCOUNT).otherwise(
```
The result is exactly the same but it would make easier to understand why the `1e-10` is there for anyone reading this code.